lib/compute/jenkins-main-node.ts

```diff
@@ -34,6 +35,11 @@ export interface JenkinsMainNodeProps extends HttpConfigProps, OidcFederateProps
   readonly sg: SecurityGroup;
 }
 
+export interface AgentNodeProps{
```

Comment:
Move to the agent node class

Reply:
I did, Forgot to remove from here. Removed now